src/main/java/atreides/ui/Ui.java

```diff
@@ -0,0 +1,47 @@
+package atreides.ui;
+
+import java.util.Scanner;
+
+/**
+ * Represents the logic for the Ui that handles the printing to user and reading user commands
+ */
+public class Ui {
+    private final String LOGO = "          _            _     _\n"
+                             +  "    / \\  | |          (_)   | |\n"
+                             +  "   /   \\ | |_ _ __ ___ _  __| | ___  ___\n"
+                             +  "  / / \\ \\| __| '__/ _ \\ |/ _` |/ _ \\/ __|\n"
+                             +  " / _____ \\ |_| | |  __/ | (_| |  __/\\__ \\\n"
+                             +  "/_/     \\_\\__|_|  \\___|_|\\__,_|\\___||___/\n";
+    private final String INTRO = "Glory to house\n" + LOGO;
+    private final String OUTRO = "Praise be Muad Dib\n" + "GoodBye\n";;
+    private final String ERROR_STRING = "File cannot be found\n";
```

Comment:
Should these variables all also be ***static***, if they are constant (screaming snake case)?